packages/connect/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.0
+
+- Update protobuf version constraint to include version `4.0.0`
```

Comment:
```suggestion
- Update protobuf version constraint to include version `0.4.0`
```

Reply:
This is about the protobuf dependency version: https://github.com/connectrpc/connect-dart/blob/8926a5eb3c3ebf956c3c1def69ce77203fd2f222/packages/connect/pubspec.yaml#L18